docs/guides/VTEX-Platform-Overview/store-architecture/store-architecture-use-cases.md

```diff
@@ -0,0 +1,163 @@
+- **Improved communication and collaboration**: Facilitate communication between users within an organization and streamline the creation of new organizations through request forms and approval queues.
+
+### Learn more
+
+- [B2B Suite - Overview](https://help.vtex.com/pt/tutorial/b2b-suite-overview--5eG6UfveWrai7looK0kVG3)
+
+## Franchise accounts (Omnichannel)
+
+This is a very common architecture model that integrates physical stores with ecommerce as [franchise accounts](https://help.vtex.com/en/tutorial/what-is-a-franchise-account--kWQC6RkFSCUFGgY5gSjdl) in VTEX.
+
+A franchise account is linked to a main account and has the following characteristics, which you can see in the reference architecture below:
+
+- **No separate website**: Franchise accounts do not have their own dedicated website, as they operate within the main account’s one, effectively acting as part of a larger [marketplace](https://help.vtex.com/pt/tutorial/o-que-e-um-marketplace--680lLJTnmEAmekcC0MIea8).
+- **Customer Data Storage**: Customer data is stored in the main account’s [Master Data](https://developers.vtex.com/docs/guides/master-data-introduction).
+- **Seller Type**: Each franchise account automatically functions as a [white label seller](https://help.vtex.com/en/tutorial/seller-white-label--5orlGHyDHGAYciQ64oEgKa) within the main account.
+- **Catalog**: Franchise accounts inherit their [catalog](https://developers.vtex.com/docs/guides/catalog-overview) from the main account.
+- **Logistics and OMS**: Each franchise account has its own [Logistics](https://developers.vtex.com/docs/guides/fulfillment) and performs its own [Order Management](https://developers.vtex.com/docs/guides/orders-overview).
+- **Prices and Payments**: Each franchise can have its own [price](https://developers.vtex.com/docs/guides/pricing-overview) and [payment](https://developers.vtex.com/docs/guides/payments-overview) methods, or you can choose to inherit them from the main account.
+- **Promotions**: It is possible to create [promotions](https://developers.vtex.com/docs/guides/promotions-overview) for each franchise account and for the main one.
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/franchises-omni.png)
+
+This architecture model works well for brands that have multiple physical stores, franchisees, or representatives. In these cases, each of them can have a franchise account in VTEX integrated into the brand’s main account. This allows them to deliver the products sold through the brand’s ecommerce from an Omnichannel perspective.
+
+### Why choose this architecture?
+
+- Potential to reduce the percentage of out-of-stock items.
+- Possible gain from inventory reduction through the use of the [Endless Aisle](https://help.vtex.com/en/tracks/estrategias-de-comercio-unificado--3WGDRRhc3vf1MJb9zGncnv/40KMlmGI5tN0r0KPCDWgGn?&utm_source=autocomplete) feature.
```

Comment:
the Endless Aisle feature = Sales App, right? Is it available outside Brazil and Latin America? If not, I think it's worth adding the disclaimer to specify the regions where this is possible at the moment